app/services/document_attributes.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class DocumentAttributes
+  TITLE_SLUG_MAX_LENGTH = 150
+  TITLE_MAX_LENGTH = 300
+  SUMMARY_MAX_LENGTH = 600
+
+  attr_reader :document
+
+  def initialize(document)
+    @document = document
+  end
+
+  def attributes(params)
+    path_title = params[:document][:title].chomp[0...TITLE_SLUG_MAX_LENGTH]
```

Comment:
We could probably do a `params.require(:document)` so we don't need to nest into params each time

Reply:
OK - you win ;-).